.circleci/config.yml

```diff
@@ -583,12 +606,17 @@ workflows:
             . venv/bin/activate
             export PATH=${PATH}:${PWD}/tests/integration/bin:/opt/vertica/bin
             export DB_FACTS_PATH=${PWD}/tests/integration/circleci-dbfacts.yml
+            echo $SCRATCH_GCS_URL  # TODO
+            echo $SCRATCH_S3_URL   # TODO
+            echo $GCP_PROJECT   # TODO
+            # TODO: Put login into other things?
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker